homeassistant/components/alexa/handlers.py

```diff
@@ -819,7 +820,7 @@ async def async_api_set_thermostat_mode(hass, config, directive, context):
 
     response = directive.response()
     await hass.services.async_call(
-        climate.DOMAIN, service, data, blocking=False, context=context
+        Platform.CLIMATE, service, data, blocking=False, context=context
```

Comment:
Services are domain-based, not platform-based.

Multiple occurrences in this PR.